source/core/GenericFunctions.cpp

```diff
@@ -668,7 +668,36 @@ InstructionCore* EmitMaxMinEltsInstruction(ClumpParseState* pInstructionBuilder)
     }
     return pInstruction;
 }
+//------------------------------------------------------------
+bool GetMinimumArrayDimensions(TypedArrayCoreRef sourceArray1, TypedArrayCoreRef sourceArray2,
```

Comment:
Please mention the list of primitives/nodes that will result in calling each of the different functions such as VectorScalar/VectorVector/Bin/UnOp just above each function. I think it might be easy as you have different tests now written and fresh in your memory.